src/styles/sidebar/components/group-list.scss

```diff
@@ -105,6 +105,16 @@ $group-list-spacing-below: 50px;
 .group-list-label__label {
   font-size: $body2-font-size;
   font-weight:bold;
+  display: inline-block;
+}
+
+.group-list-label__toggle {
+  color: $gray-dark;
+  cursor: default;
+}
+
+.has-toggle group-list-label__toggle {
```

Comment:
To match both these classes on the same element, this should be `.group-list-label__toggle.has-toggle` (the lack of spacing is the important part). Otherwise this becomes a descendant selector ("any child of a `has-toggle` element, include itself, that has `group-list-label__toggle`).